app/models/book.rb

```diff
@@ -19,6 +17,10 @@ def to_s
     slug
   end
 
+  def discussions_in_organization(organization = Organization.current)
```

Comment:
This one is odd. It will be addressed in another PR.
The controller should be organization_discussions instead of book_discussions, as it doesn't make sense now.
And this line could be removed.